README.md

```diff
@@ -276,9 +290,16 @@ The project wiki is "public" and contributions there are also welcome.
 
 Before raising a new issue, please search previous issues (both open and
 closed), to see if your question has already been answered or problem resolved.
-If reporting a problem, please clearly state the version of BlueALSA that you
-are using, and give sufficient information for readers to be able to reproduce
-the issue.
+If reporting a problem, please clearly state:
+
+* the version of BlueALSA that you are using (`bluealsa --version`),
```

Comment:
I'll try to tinker a little bit with issue and pull request templates. It will be better to provide such information as a context note. For PR template it's nice to have something like that:

```
> !!!!!!!!!! Instructions
>
> Please have a good description of the problem and the fix. Help the reviewer
> understand what to expect.
>
> For new feature, extensive change or non-trivial bug fix add a simple unit test
> or at least describe how to test the code manually.
>
> If you have an issue number, please reference it with a syntax `Fixes #123`.
>
> Please delete these instructions to prove you have read them.
>
> !!!!!!!!!! Instructions End
```